htdocs/product/stock/.htaccess

```diff
@@ -0,0 +1,7 @@
+RewriteEngine On
```

Comment:
Adding  files with specific setup to an external software and environment variable is not portable.
Can you remove this file.